source/css/scss/app/pages/checkout.scss

```diff
@@ -61,14 +65,84 @@
             }
             @media #{$large-up} {
                 .loan-details {
-                    padding: inherit;
+                    padding: 0 .8rem;
                 }                
             }
+
+            .loan-totals {
+                .amount-select {
+                    float: right;
+                    direction: rtl;
```

Comment:
The text isn't actually read right-to-left is it? if it needs to be right-aligned use text-align: right;

Reply:
Had to keep this direction: rtl to make the select align right. Cleaned out the other which was on an input and not needed.